TABLE.md

```diff
@@ -0,0 +1,25 @@
+# Built-in Decorators Synopsis
+
+|                           |                              `@wrap(f)`                                   |                                                       `@register(f)`                                                         |             `@expose(f)`                                                                                                     |             `@initialize(f)`          |
+|---------------------------|---------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------    |------------------------------------------------------------------------------------------------------------------------------|---------------------------------------|
+| **when**                  | while setting up the class                                                | after the class is created                                                                                                   | after the class is created                                                                                                   | after executing the class initializer |
```

Comment:
The term "class initializer" is a bit ambiguous. I like to refer to this as "field initializer".